src/main/resources/cloudformation/vpc.yaml

```diff
@@ -21,6 +22,9 @@ Outputs:
   vpcSubnetIdForAz3:
     Description: The VPC subnet in AZ '3'
     Value: !Ref 'CerberusSubnetForAz3'
+  vpcSubnetIdForAz4:
```

Comment:
Should be conditional

Reply:
The condition is used in the "Value" field

Reply:
Does it output an empty string?

Reply:
Yes, if a value for the az4 parameter is not provided it outputs an empty string.

Reply:
Yes, if a value for the az4 parameter is not provided it outputs an empty string.